### DISCUSSION TITLE
Unable to start supabase

### DISCUSSION THREAD

**Discussion by nandox5** [supabase/supabase#4436] [Questions] [answered]
I am trying to install supabase, but i run into a problem when running docker-compose up

```
docker-compose : kong Pulling 
At line:1 char:1
+ docker-compose  up
+ ~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (kong Pulling :String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
auth Pulling 
rest Pulling 
db Pulling 
meta Pulling 
studio Pulling 
storage Pulling 
realtime Pulling 
storage Error 
meta Error 
db Error 
kong Error 
realtime Error 
no matching manifest for windows/amd64 10.0.14393 in the manifest list entries
```

Any ideas?

Docker --version : 20.10.7
Docker-compose --version: 2.2.2
OS: Windows server 2016

**Comment by soedirgo** [comment, marked as answer]
It's only available for linux/amd64 - on Windows you can use WSL or some other way to run it.

**Reply by jtkeyva** [reply]
@soedirgo Any news when M1 will be supported?

**Comment by jtkeyva** [comment]
I finally manage to get https://supabase.com/docs/guides/cli/local-development working on my M1. I had to crank up the RAM in Docker (I went all the way to 16GB) and now it works even though it's in emulation mode.
